client/src/component/User.js

```diff
@@ -1,10 +1,29 @@
+import React, { useState } from "react";
+import "./User.css";
+
+
 export function User() {
 
+	const [name, setName] = useState("");
+
+	const handleInputChange = (event) => {
+		setName(event.target.value);
+	};
+
```

Comment:
thank you for writing this , but I am not sure we will supused to write functionality yet or not ?

Reply:
It doesn't break the code.

Reply:
no it doesn;t but maybe later will be change